src/main/java/com/melon/portfoliomanager/models/PortfolioItem.java

```diff
@@ -0,0 +1,76 @@
+package com.melon.portfoliomanager.models;
+
+import jakarta.persistence.*;
+
+
+@Entity
+@Table(name = "portfolio_items")
+public class PortfolioItem {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
```

Comment:
Why do you use this mapping here ? Have not we agreed not to use it at this point of development ?

Reply:
fixed that.